### DISCUSSION TITLE
How to access phpmyadmin and create new joomla site

### DISCUSSION THREAD

**Discussion by VanShane** [joomlatools/joomlatools-pages-server#56] [Q&A] [answered]
Hi, 

So I managed to run joomlaserver and I'm able to access the default site, I was wondering how to add aditionnal sites to the site directory and how to access phpmyadmin, if I understand it right I should be able to do so on http://localhost:8180/phpmyadmin.

I also noticed the default site doesn't go trough the install process so there is a default user existing but I can't find any information on that in the project.

Thx in advance !

**Comment by johanjanssens** [comment, marked as answer]
Hello Shane,

### phpMyAdmin

Great to hear that. Joomatools Server comes with additional Docker Compose profiles: https://docs.docker.com/compose/profiles, phpMyAdmin is one such profile. To start it you would do:

`docker composer --profile phpmyadmin up`

This will start both the `joomlatools-server` main container and an additional `phpmyadmin` container, both are setup for you to access the same database, by default data is stored in a Docker volume.

### Creating sites

Creating additional sites is done using our Joomlatools Console which comes included. To access it you need to open a shell into the container, using `docker exec -it joomlatools-server  bash` and then you can use `joomla` 

For more info: https://github.com/joomlatools/joomlatools-console 

A new v2 version of Console us underway, we are adding Joomla 4 support and streamlining the integration into Server. See: https://github.com/joomlatools/joomlatools-console/pull/148

**Reply by VanShane** [reply]
Thank you Johan for your answer, I'll check it out !

**Reply by johanjanssens** [reply]
You are most welcome. We just added support for dnsmasq today: https://github.com/joomlatools/joomlatools-server/pull/58 In short it allows you to run custom local .test domains. Each site gets it's own domain, makes it really easy to test sites properly without needing to use subfolders. More info in the PR.
